packages/suite/src/services/suite/metadata/FileSystemProvider.ts

```diff
@@ -47,6 +47,22 @@ class FileSystemProvider extends AbstractMetadataProvider {
         return this.ok(undefined);
     }
 
+    async getMetadataFilesList() {
+        const response = await desktopApi.getMetadataFiles();
+
+        if (!response.success) {
+            return this.error('PROVIDER_ERROR', response.error);
+        }
+
+        // don't return the `DS_Store` folder on macOS
+        const index = response.payload.indexOf('.DS_Store');
```

Comment:
shouldn't this be handled in node layer? I would maybe exclude all hidden files by default. If someone needs them in future, we could add a parameter that returns them too

Reply:
fixed